samples/bluetooth/fast_pair/locator_tag/src/ui/platform_dk.c

```diff
@@ -139,12 +144,14 @@ static void run_led_thread_process(void)
 	static bool run_led_on = true;
 
 	while (1) {
-		if (atomic_get(&run_led_thread_state_app_running)) {
+		uint32_t sleep_time_ms = RUN_LED_BLINK_INTERVAL_MS;
+
+		if (atomic_test_bit(ui_state_status, APP_UI_STATE_APP_RUNNING)) {
 			dk_set_led(APP_RUN_STATUS_LED, run_led_on);
 			run_led_on = !run_led_on;
```

Comment:
Can we first update `run_led_on` and then call `dk_set_led` (so that value set matches RAM content)?